magicgui/widgets/_bases.py

```diff
@@ -293,28 +294,6 @@ def native(self):
         """Return native backend widget."""
         return self._widget._mgui_get_native_widget()
 
-    def show(self, run=False):
```

Comment:
most of this is just moved in this PR... only new things is the `tooltip` property